sdk/search/azure-search-documents/tests/async_tests/recordings/test_search_index_client_skillset_live_async.test_create_skillset.yaml

```diff
@@ -1,45 +1,46 @@
 interactions:
 - request:
     body: '{"name": "test-ss", "description": "desc", "skills": [{"@odata.type": "#Microsoft.Skills.Text.EntityRecognitionSkill",
-      "inputs": [{"name": "text", "source": "/document/content"}], "outputs": [{"name":
-      "organizations", "targetName": "organizationsS1"}], "includeTypelessEntities":
-      true}, {"@odata.type": "#Microsoft.Skills.Text.V3.EntityRecognitionSkill", "inputs":
-      [{"name": "text", "source": "/document/content"}], "outputs": [{"name": "organizations",
-      "targetName": "organizationsS2"}], "modelVersion": "3"}, {"@odata.type": "#Microsoft.Skills.Text.SentimentSkill",
+      "name": "skill1", "inputs": [{"name": "text", "source": "/document/content"}],
```

Comment:
Curious why we changed the request payload and added "name"? Will this cause problem?

Reply:
Let me rephrase my question, now we added "name" in the payload, is it required? If yes, why it worked before w/o it? Is it a breaking change?

Reply:
You can only reset skills that you have given a name. Hence I had to add the names in the test to be able to test `reset_skills`.

Reply:
To make sure I understand, it is allowed to create a no-named skill. But user will not be able to reset it, right?

Reply:
Correct.